sdk/client_new_method_params_context/src/models.rs

```diff
@@ -10,37 +10,35 @@ pub struct Secret {
     pub version: String,
     #[serde(rename = "attributes")]
     pub properties: SecretProperties,
-    #[serde(skip)]
-    pub etag: Option<Etag>,
 }
 
 impl TryFrom<Response<Secret>> for Secret {
     type Error = azure_core::Error;
 
     fn try_from(value: Response<Secret>) -> Result<Self, Self::Error> {
-        let etag: Option<Etag> = value.headers().get_optional_str(&ETAG).map(Etag::from);
-
         let f = || value.into_body().json::<Secret>();
-        let mut secret = block_on(f())?;
-
-        secret.etag = etag;
-        Ok(secret)
+        block_on(f())
```

Comment:
Should we actually use something like `spawn_block` if we can avoid a dependency on a specific async runtime?